src/main/java/de/unistuttgart/iste/sqa/pse/sheet11/presence/calculator/Calculator.java

```diff
@@ -91,22 +88,20 @@
 	 */
 	public int addExact(final int summand1, final int summand2);
 
-	/*
-	 * @requires value != Integer.MIN_Value
-	 * 
-	 * @ensures value >= 0 ==> \result == value
-	 * 
-	 * @ensures value < 0 ==> \result == value*-1
-	 * 
-	 * 
-	 */
+	/*@
+	 @ requires value != Integer.MIN_Value;
+	 @
+	 @ ensures value >= 0 ==> \result == value;
+	 @ ensures value < 0 ==> \result == -1 * value;
+	 @
+	 @*/
 	/**
 	 * calculates absolute value of a given integer value
 	 * 
 	 * if the value is equal to minimal Integer Value a Arithmetic Exception is
 	 * thrown because Integer.MIN_Value is not displayable as a positive number
 	 * 
-	 * @param integer value
+	 * @param value value
 	 * @return absolute value
```

Comment:
Can we specify param and return a bit?

Reply:
I removed the @param because it probably falls under 'Pflichtkommentare' which should be avoided according to the lecture, so we shouldn't do it ourselves?